src/display/views/threads/TheirTurnThreads.js

```diff
@@ -28,13 +30,15 @@ function mapStateToProps(state) {
 	const lastPosters = selectors.getActiveThreadLastPosters(state);
 	const tags = selectors.getActiveThreadTags(state);
 	const filteredThreads = selectors.getTheirTurnFilteredThreads(state);
+	const { useLightTheme } = getUi(state);
```

Comment:
Correct me if I'm wrong, but I don't think using the selector here nets us anything. `const { useLightTheme } = state.ui` is a straight property access which if anything should be a smidge more efficient.

Reply:
The only benefit is that it keeps the state structure opaque. If the `ui` slice gets moved around or renamed and we just use property access, all the mapStateToProps consumers would need to be adjusted.

Reply:
Nope, this is a good idea; I like it. And, judging by a little googling, a more ubiquitous practice than I realized. :O I've learned something today. 

May make this adjustment elsewhere in the codebase too/use it going forward.